core/ProfessionalServices/Advertising.php

```diff
@@ -117,7 +117,10 @@ private function getCampaignParametersForPromoUrl($campaignName, $campaignMedium
      */
     public static function isAdsEnabledInConfig($configGeneralSection)
     {
-        $oldSettingValue = @$configGeneralSection['piwik_pro_ads_enabled'];
+        $oldSettingValue = false;
+        if (isset($configGeneralSection['piwik_pro_ads_enabled'])) {
+            $oldSettingValue = @$configGeneralSection['piwik_pro_ads_enabled'];
```

Comment:
Do we still need the `@` here?

Reply:
I suppose technically not but might also not hurt?

Reply:
seems redundant, but ok